includes/modules/payment/paypal/paypalwpp_admin_notification.php

```diff
@@ -123,20 +123,22 @@
 
         $outputPFmain .= '</table></td>'."\n\n";
     }
+} else if ($response == false) {
```

Comment:
This `==` is probably best as `===` since we're testing for boolean vs the usual array.